docs/noise.ipynb

```diff
@@ -62,196 +62,673 @@
     "</table>"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "846b32703c5c"
+   },
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import cirq\n",
+    "except ImportError:\n",
+    "    print(\"installing cirq...\")\n",
+    "    !pip install --quiet cirq\n",
+    "    print(\"installed cirq.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "c3c7725dfc15"
+   },
+   "outputs": [],
+   "source": [
+    "import cirq"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {
     "id": "7d3daa944d6b"
    },
    "source": [
-    "For simulation, it is useful to have `Gate` objects that enact noisy quantum evolution.\n",
-    "\n",
-    "Cirq supports modeling noise via *operator sum* representations of\n",
-    "noise (these evolutions are also known as quantum operations, quantum\n",
-    "dynamical maps, or superoperators).  \n",
+    "For simulation, it is useful to have `Gate` objects that enact noisy quantum evolution. Cirq supports modeling noise via *operator sum* representations of noise (these evolutions are also known as quantum operations, quantum dynamical maps, or superoperators). \n",
     "\n",
-    "This formalism models evolution of the\n",
-    "density matrix via:\n",
+    "This formalism models evolution of the density matrix $\\rho$ via\n",
     "\n",
-    "$$\\rho \\rightarrow \\sum_k A_k \\rho A_k^\\dagger$$\n",
+    "$$\n",
+    "\\rho \\rightarrow \\sum_{k = 1}^{m} A_k \\rho A_k^\\dagger\n",
+    "$$\n",
     "\n",
-    "Where A<sub>k</sub> are *Kraus* operators. These operators are not\n",
-    "necessarily unitary and must satisfy the trace-preserving property:\n",
+    "where $A_k$ are known as *Kraus operators*. These operators are not necessarily unitary and must satisfy the trace-preserving property\n",
```

Comment:
and -> but?

Reply:
Yeah I think that sounds better. I guess it was "and" originally because they are two independent criteria, but "but" sounds better with the qualifier "not necessarily unitary." English is hard.